File: AppCurrencyAndroid/appforyou/app/src/main/java/com/allerates/appforyou/fragment/MySupportMapFragment.java
package com.allerates.appforyou.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.allerates.appforyou.widget.TouchableWrapper;
import com.google.android.gms.maps.SupportMapFragment;

/**
 * Created by Rikmen on 26.06.2018.
 */

public class MySupportMapFragment extends SupportMapFragment {
    public View mOriginalContentView;
    public TouchableWrapper mTouchView;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup parent, Bundle savedInstanceState) {
        try{
        mOriginalContentView = super.onCreateView(inflater, parent, savedInstanceState);
        mTouchView = new TouchableWrapper(getActivity());
        mTouchView.addView(mOriginalContentView);
        return mTouchView;
        } catch (Exception e) {
            return null;
        }
    }

    @Override
    public View getView() {
        return mOriginalContentView;
    }
}

File: AppCurrencyAndroid/app/src/main/java/com/allerates/appforyou/fragment/MySupportMapFragment.kt
package com.allerates.appforyou.fragment


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.allerates.appforyou.widget.TouchableWrapper
import com.google.android.gms.maps.SupportMapFragment


class MySupportMapFragment:
        SupportMapFragment() {

    var mOriginalContentView: View? = null
    lateinit var mTouchView: TouchableWrapper

    override fun onCreateView(inflater: LayoutInflater, parent: ViewGroup?, savedInstanceState: Bundle?): View? {
        return try {
            mOriginalContentView = super.onCreateView(inflater, parent, savedInstanceState)
            mTouchView = TouchableWrapper(activity)
            mTouchView.addView(mOriginalContentView)
            mTouchView
        } catch (e: Exception) {
            null
        }

    }

    override fun getView() = mOriginalContentView
}
